13MD05-90_changelog.md

```diff
@@ -1,5 +1,52 @@
 # Changelog for 13MD05-90 "MEN MDIS for Linux System Package plus lowlevel drivers"
 
+## [13MD05-90_02_05] - 2023-05-01
+
+### Added
+- Multiple improvements in functional and compilation testing tool
+- Automatic tests for test setup 7
+- F27 and G28 CPU Boards support
+- scan_system.sh: Detection of CPU Board G23
+- scan_system.sh: Detection of new FPGA board F401
+- 13Z072-06: OWB enhancements
+- 13Z077-90: Operstate functionality
+- 13Z055-90: Half duplex support
+- 13Z055-90: Send one/four start flag(s)
+- fpga_load: read chameleon table from IO-MAPPED memory regions
+- Cross compilation support
```

Comment:
Maybe my question was confusing. ;-)
I'm just wondering about it, because general cross compilation supported is already available since ages.
Is this note related to https://github.com/MEN-Mikro-Elektronik/13MD05-90/issues/246? Than it should be moved to the Fixed section.

Reply:
Yes, this was part of the fix of #246. We just added the prefix for the cross compilation toolchain.

Reply:
Perfect. I just uploaded it again